cli/image-scanning-by-using-the-roxctl-cli.adoc

```diff
@@ -0,0 +1,71 @@
+[id="image-scanning-by-using-the-roxctl-cli"]
+:_content-type: ASSEMBLY
+= Image scanning by using the roxctl CLI
+include::modules/common-attributes.adoc[]
+:context: image-scanning-by-using-the-roxctl-cli
+
+toc::[]
+
+[role="_abstract"]
+You can use `roxctl` CLI, Jenkins, and API to scan images stored in a cluster specific {ocp} integrated image registry. To do so, specify the appropriate cluster using the delegated scanning configuration or the cluster parameter in `roxctl` CLI, Jenkins, and API.
+
+[id="scanning-images-in-a-remote-cluster_{context}"]
+== Scanning images in a remote cluster
+
+To scan an image in a remote cluster, use the `roxctl image scan` command. Specify the location and name of the image, and specify the remote cluster. You can also specify optional parameters to customize the behavior of the command.
+
+.Procedure
+
+* Run the following command to scan the specified image in a remote cluster:
++
+[source,terminal,subs="+quotes"]
+----
+$ roxctl image scan --image=_<image_registry>_/_<image_name>_ --cluster=_<cluster_name>_ [flags]<1>
+----
++
+--
+<1> For `<image_registry>`, specify the registry where the image is located, for example, `image-registry.openshift-image-registry.svc:5000/`.
++
+For `<image_name>`, specify the name of the image you want to scan, for example, `dave/dave-is:latest`.
```

Comment:
May be use a different image name instead of `dave/dave-is:latest`, something like `default/image-stream:latest` may be easier to digest.